src/main/java/com/projecturanus/betterp2p/client/gui/GuiRenameItem.kt

```diff
@@ -0,0 +1,39 @@
+package com.projecturanus.betterp2p.client.gui
+
+import appeng.client.gui.widgets.MEGuiTextField
+import com.projecturanus.betterp2p.MODID
+import com.projecturanus.betterp2p.client.TextureBound
+import net.minecraft.client.gui.GuiScreen
+import net.minecraft.util.ResourceLocation
+
+class GuiRenameItem(msg: InfoWrapper): GuiScreen(), TextureBound {
```

Comment:
what this gui for?

Reply:
This is automatically generated. I forget delete it